### DISCUSSION TITLE
[Expression-Language] How would I access an object field containing a space in the field name?

### DISCUSSION THREAD

**Discussion by keevan** [symfony/symfony#47223] [Q&A]
In the working with object section, it illustrates how you can access object fields/properties using the dot notation. 
https://symfony.com/doc/current/components/expression_language/syntax.html#working-with-objects

However, if I had a PHP object `\stdClass`, with a field names containing spaces, how can I access it using dot notation? 
```
$object = new \stdClass;
$object->{'field with space'} = "value";
```

However, I am able to access it if it were an array, using the format `$object["field with space"]` based on https://symfony.com/doc/current/components/expression_language/syntax.html#working-with-arrays

Is there a way to do this without first converting the object into an array?

**Comment by lyrixx** [comment]
I'm not sure, but I think it's not possible
